src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -2445,10 +2443,6 @@ def sort_index(
             ErrorMessage.not_implemented(
```

Comment:
where is inplace actually handled here? To me this PR seems to simply remove the error, but I am not sure where the inplace param actually takes effect?